actionlist/app/actions/action_items.js

```diff
@@ -1,8 +1,6 @@
 import { createAction } from 'redux-actions';
-import { TOGGLE_ITEM } from './types';
+import { TOGGLE_ITEM, TOGGLE_DRAWER } from './types';
```

Comment:
Imports should generally be in alphabetical order by file and when importing multiple modules from the same import, those modules should also be alphabetical.